SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -197,28 +198,35 @@ Quarks {
 			},
 			prev = this.installed.detect({ |q| q.name == quark.name });
 
-		if(prev.notNil and: {prev.localPath != quark.localPath}, {
+		if(prev.notNil and: {prev.localPath != quark.localPath}) {
```

Comment:
I know this wasn't your change, but since you're editing here, spaces pad enclosed curly brackets:
```supercollider
if(prev.notNil and: { prev.localPath != quark.localPath }) { ...
```

Reply:
changed - thanks! so long as we are making style changes I also changed the lone `arg` to pipes

Reply:
Great!